addons/account/__init__.py

```diff
@@ -25,16 +26,15 @@ def _auto_install_l10n(cr, registry):
         elif country_code == 'DE':
             module_list.append('l10n_de_skr03')
             module_list.append('l10n_de_skr04')
-        else:
+        elif not to_install_l10n:
```

Comment:
@william-andre It does not make sense that the behaviour is different when you are in the syscohada, ...  Under to_install_l10n you should immediately have if not to_install_l10n:

Nice-to-have: Then it would also make sense to change the country of the company on installation of a certain CoA.  (maybe if it has not been set already)

Reply:
I don't understand your first statement.

Regarding the Nice-to-have, it is difficult because of things like syscohada... If we don't have `l10n_bj`, `l10n_bf`,... I don't see how we can do this.

Reply:
Did you want something like
```python3
if to_install_l10n:
    # We don't look install a CoA if one was passed in the command line
    # or has been selected to install
    pass
elif country_code in SYSCOHADA_LIST:
    #countries using OHADA Chart of Accounts
    module_list.append('l10n_syscohada')
elif country_code == 'GB':
    ....
```

Reply:
@william-andre Then we should indeed have a country_id on the CoA or not for the nice-to-have.  It is indeed not that straightforward. 

An if with a pass is a bit weird.  I would have said if not to_install: and then put everything a level deeper.

Reply:
More diff, and that is really subjective. I like not having too much levels of indentation in the code, and it makes sense in a `if ... elif ... else` scenario.
I did it this time, but I might be more reluctant in the future :)